packages/ui/src/components/select.v2.tsx

```diff
@@ -65,21 +107,6 @@ export function Select({
   );
 }
 
-export function SelectItem({ className, children, ...props }: SelectItemProps) {
-  return (
-    <Item
-      className={cx(
-        'flex select-none items-center gap-2 rounded-md p-2 text-sm',
-        'data-[highlighted]:bg-primary data-[highlighted]:text-white data-[highlighted]:outline-none',
-        'disabled:pointer-events-none disabled:opacity-50',
-        className
-      )}
-      {...props}
-    >
-      <ItemText>{children}</ItemText>
-      <ItemIndicator>
-        <Check size={16} />
-      </ItemIndicator>
-    </Item>
-  );
-}
```

Comment:
Let's keep the existing structure, no need to make this a compound component (ie: `Select.Option`), though I'm okay with you renaming this to `Option` if you like that better than `SelectItem`!